src/skins/vector/css/matrix-react-sdk/structures/RoomView.css

```diff
@@ -202,6 +202,25 @@ hr.mx_RoomView_myReadMarker {
     width: 100%;
     -webkit-flex: 0 0 auto;
     flex: 0 0 auto;
+
+    max-height: 0px;
+    background-color: #fff;
```

Comment:
Is there something I might have to merge in to get that to work? I've got `npm start:css` running, but it doesn't seem to be doing SCSS